templates/user/edit-profile.html

```diff
@@ -85,7 +98,17 @@
             $('#disable-2fa-button').click(function () {
                 alert("The administrators for this site require all the staff to have Two-factor Authentication enabled, so it may not be disabled at this time.");
             });
-
+        });
+    </script>
+    <script type="text/javascript">
+        $(function () {
+            $('#generate-scratch-codes-button').click(function() {
+                $('#scratch-codes-text').css({"font-family": "Segoe UI, Lucida Grande, Arial, sans-serif"});
```

Comment:
This is a bit of a hack... can't we put this in another div instead?

Reply:
Another div...which replaces the current `scratch-codes-text` on click?